### DISCUSSION TITLE
Проблема с импортом .level

### DISCUSSION THREAD

**Discussion by Nicel193** [PavelBlend/blender-xray#440] [Q&A] [answered]
При попытке импорта файла с расширение .level выдает ошибку
![Снимок экрана 2021-10-27 140114](https://user-images.githubusercontent.com/64494378/139053558-c220b73e-1efd-42e3-952a-815a4214fd32.png)

**Comment by PavelBlend** [comment, marked as answer]
@Nicel193 скорее всего ты импортируешь не тот *.level
Аддон поддерживает Scene Selection - это выделение из сдк. Но не поддерживает обычные уровни сцены из level editor.
Вот здесь тот же вопрос:
https://ap-pro.ru/forums/topic/87-blender/page/5/?tab=comments#comment-62358
И мой ответ:
https://ap-pro.ru/forums/topic/87-blender/page/5/?tab=comments#comment-62380

**Reply by Nicel193** [reply]
Спасибо за ответ, помогло!
